src/controllers/private/user-images-controller.js

```diff
@@ -1,27 +1,43 @@
 const HSC = require('http-status-codes');
-const userImagesService = require('../../services/index').userImagesService;
+const imagesService = require('../../services/index').imagesService;
 
 module.exports = {
   async addNewImage(req, res) {
-    return res.json(await userImagesService.addNewImage(req));
+    return res.json(await imagesService.create(req));
   },
 
   async getUserImages(req, res, next) {
-    const result = await userImagesService.getUserImages(req);
+    const result = await imagesService.getAllByUserId(req);
 
     if (!result.length) return next();
 
     return res.json(result);
   },
+  async getUserImagesPagination(req, res, next) {
```

Comment:
Why this func named "getUserImages**Pagination**" ? This func still returns a list of images as main part of result.